wordproof-timestamp/includes/Page/SettingsPage.php

```diff
@@ -37,18 +30,34 @@ public function generateSettingsPage() {
         wp_localize_script('wordproof.admin.js', 'wordproofSettings', [
             'network' => get_option('wordproof_network', false),
             'certificateText' => CertificateHelper::getCertificateText(),
-            'saveChanges' => __('Save Changes')
+            'saveChanges' => __('Save Changes', 'wordproof-timestamp')
         ]);
 
         ?>
         <div class="wrap">
-            <h1>WordProof Settings</h1>
-            <form method="post" action="options.php">
-                <?php settings_fields( 'wordproof-options' ); ?>
-                <?php do_settings_sections( 'wordproof-options' ); ?>
+            <h1><?php __( 'WordProof Settings', 'wordproof-timestamp' ); ?></h1>
```

Comment:
I see `'wordproof-timestamp'` being used quite often. Maybe make a constant out of this or wrap the function `__`?